src/lib/autoinstall/clients/inst_autoinit.rb

```diff
@@ -278,7 +278,7 @@ def ProfileSourceDialog(original)
           if ret == :abort && Yast::Popup.ConfirmAbort(:painless)
             break
           elsif ret == :retry
-            uri = Yast::Convert.to_string(UI.QueryWidget(Id(:uri), :Value))
+            uri = Yast::Convert.to_string(Yast::UI::QueryWidget(Id(:uri), :Value))
```

Comment:
please do not use :: for calling method. It should be `Yast::UI.QueryWidget`. rubocop should complain for it.

Reply:
Fixed. Hm, rubocop has not blamed it.